frontend/src/components/ChartProductionData.jsx

```diff
@@ -271,7 +271,7 @@ const ChartProductionData = () => {
           value={line ? 1 : 0}
           exclusive
           onChange={(event, value) => {
-            setLine(!!value)
+            value !== null && setLine(!!value)
```

Comment:
There is no need to add complexity to the logic (that should be removed when bars are available) if we just disable both buttons, meanwhile.